### ISSUE TITLE
build project error

### ISSUE THREAD

**Issue by MagicZer0** [frohoff/ysoserial#146]
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /root/tmp/ysoserial/src/main/java/ysoserial/payloads/FileUpload1.java:[11,35] error: cannot find symbol
[ERROR]   symbol:   class DeferredFileOutputStream
  location: package org.apache.commons.io.output
/root/tmp/ysoserial/src/main/java/ysoserial/payloads/FileUpload1.java:[12,35] error: cannot find symbol
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.761 s
[INFO] Finished at: 2020-11-29T12:01:23+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project ysoserial: Compilation failure: Compilation failure: 
[ERROR] /root/tmp/ysoserial/src/main/java/ysoserial/payloads/FileUpload1.java:[11,35] error: cannot find symbol
[ERROR]   symbol:   class DeferredFileOutputStream
[ERROR]   location: package org.apache.commons.io.output
[ERROR] /root/tmp/ysoserial/src/main/java/ysoserial/payloads/FileUpload1.java:[12,35] error: cannot find symbol
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

add maven dependency for commons-io:
```
<dependency>
    <groupId>commons-io</groupId>
    <artifactId>commons-io</artifactId>
    <version>2.4</version>
</dependency>
```

**Comment by gothburz** [comment]
Thanks @MagicZer0 was running into this issue as well, adding commons-io dependency to pom.xml resolved for me.

**Comment by IanSmith123** [comment]
Thanks @MagicZer0 , this work for me.

**Comment by Alexandre-Bartel** [comment]
See PR #148

**Comment by frohoff** [comment]
Closed after #148 merged.
